.github/workflows/continuous_integration.yml

```diff
@@ -17,32 +17,25 @@ jobs:
     - name: ⤵️ Checkout Source
       uses: actions/checkout@v3
 
-    - name: 🛠️ Setup .NET Core SDK 3.1.x
+    - name: 🛠️ Setup .NET 
       uses: actions/setup-dotnet@v3
       with:
-        dotnet-version: '3.1.x'
-
-    - name: 🛠️ Setup .NET Core SDK 6.0.x
-      uses: actions/setup-dotnet@v3
-      with:
-        dotnet-version: '6.0.x'
-
-    - name: 🛠️ Setup .NET Core SDK 7.0.x
-      uses: actions/setup-dotnet@v3
-      with:
-        dotnet-version: '7.0.x'
+        dotnet-version: | 
+            6.0.x
+            7.0.x
```

Comment:
Do we need this now, since it is removed as a target and testrun ?

Reply:
Removed.